CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased](https://github.com/pusher/chatkit-server-node/compare/1.3.0...HEAD)
```

Comment:
This should be a major version bump right? Or are you going to do this when you release the library?

Reply:
It certainly should be yeah, will bump it when I release.